### PR TITLE
Splunk Mirroring Bugfix

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/36653

## Description
Returning the delta changed and not the whole notable in `get-remote-data` command.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Review by ShahafBenYakir — COMMENTED** [review]
let's send this to Josh and document that the only fields we are mirroring are: 'comment', 'status', 'owner', 'urgency'

**Comment by anara123** [comment]
@guyfreund please open/link the corresponding issue

**Comment by guyfreund** [comment]
> @guyfreund please open/link the corresponding issue

@anara123 done

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py [hunk 1]
```diff
@@ -43,6 +43,8 @@
     'Outgoing': 'Out',
     'Incoming And Outgoing': 'Both'
 }
+OUTGOING_MIRRORED_FIELDS = ['comment', 'status', 'owner', 'urgency']
```
Comment (ShahafBenYakir) [inline Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:46]: In our outgoing mapper are we mapping status also to 'status_label'?

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py [hunk 2]
```diff
@@ -993,21 +995,26 @@ def get_remote_data_command(service, args, close_incident):
...
+                'Contents': {
+                    'dbotIncidentClose': True,
+                    'closeReason': 'Notable event was closed on Splunk.'
+                },
+                'ContentsFormat': EntryFormat.JSON
+            })
+
+        demisto.debug('Updated notable {}'.format(notable_id))
+    else:
+        demisto.debug('no delta was found for notable {}'.format(notable_id))
```
Comment (ShahafBenYakir) [inline Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:1015]: Is this intentionally outside the for loop?

Comment (guyfreund) [reply Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:1015]: yes, the loop is just to get the most update version of the notable

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py (hunk 1 above)
Comment (guyfreund) [reply Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:46]: nope, we don't use it that way inn the new outgoing mapper

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py (hunk 2 above)
Comment (ShahafBenYakir) [reply Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:1015]: So why do you need `entries` to be a list?

File: Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py:46]: So we are telling users we are able to mirror in the `status_label` but not out?

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**